srv/ChangeCameras.srv

```diff
@@ -1,5 +1,3 @@
-bool primary # true if requesting streams for primary laptop
-CameraCmd[4] camera_cmds
+CameraCmd camera_cmds
```

Comment:
Sorry about the changes but...

Here's the link to the finalized ChangeCameras.srv:
https://github.com/umrover/mrover-ros/blob/justiny/cameras/srv/ChangeCameras.srv